.github/workflows/test_pr_and_main.yml

```diff
@@ -113,14 +113,14 @@ jobs:
           PACKAGES: cython
 
         - os: windows-latest
-          python: 3.9
+          python: '3.10'
```

Comment:
... Let me guess. `3.9` is fine, `3.11` is fine, but `'3.10'` has to have strings?

Reply:
Yup - GHA interprets things that look like numbers as numbers unless you explicitly make them strings.  Without the quotes, 3.10 looks like (and becomes) 3.1.

Reply:
_of course_